### ISSUE TITLE
install script bug when using _ and or - in db name

### ISSUE THREAD

**Issue by spacecabbie** [chamilo/chamilo-lms#3667]
**Describe**
Install error when using - in db name or when using it in combo with _:
Ex: my_db-test

**To Reproduce**
Download latest release 
and walk through install script.

**Expected behavior**
Eighter error message that name is not alowed or to recognise the name of th db now it only sees -test and thus fails install 

**Screenshots**
https://i.imgur.com/YjV30ZN.png

**Comment by ywarnier** [comment]
Yes, I think we have reported this in the past in an issue. Not a priority for us at the moment, but thanks for reporting. In the meantime, please use a non "_/-" database name :-)

**Comment by spacecabbie** [comment]
> Yes, I think we have reported this in the past in an issue. Not a priority for us at the moment, but thanks for reporting. In the meantime, please use a non "_/-" database name :-)

Of course it's a minor thing but am getting the hang of the system more and more so this is also a reminder to self of possible small issues i might be able to fix and that way help out.

**Comment by christianbeeznest** [comment]
It is checked and improved with this commit https://github.com/chamilo/chamilo-lms/pull/7058/commits/4487dbd1aa2a5b55db22239fc1dcb3495e6028c6

Thanks for confirmation.

**Comment by ywarnier** [comment]
Tried locally with user "cha-test", db "cha_test" and password "cha-_+/test1;" and it all worked (created 315 tables, let me login, etc).
Thanks!
